File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/datadriven/DataDrivenViewModel.cpp
#include "DataDrivenViewModel.h"
#include <vector>
#include <memory>
#include "DataDrivenViewModelResultRow.h"
#include <string>

namespace datadriven
{
  std::vector<std::shared_ptr<DataDrivenViewModelResultRow>> DataDrivenViewModel::getResultTableRows()
  {
    return this->ResultTableRows;
  }
  void DataDrivenViewModel::loadViewModel(std::string name)
  {
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/datadriven/DataDrivenViewModel.cs
/// <filename>
///     DataDrivenViewModel.cs
/// </filename>
using System.Collections.Generic;
using datadriven;

namespace datadriven
{
    public class DataDrivenViewModel
    {
        private System.Collections.Generic.List<datadriven.DataDrivenViewModelResultRow> ResultTableRows = new System.Collections.Generic.List<datadriven.DataDrivenViewModelResultRow>();

        public virtual System.Collections.Generic.List<datadriven.DataDrivenViewModelResultRow> getResultTableRows()
        {
            return this.ResultTableRows;
        }

        public virtual void loadViewModel(string name)
        {
        }
    }
}
